website/versioned_docs/version-0.14.1/docker_demo.md

```diff
@@ -134,9 +135,10 @@ $ docker ps
 
 :::note Please note the following for Mac AArch64 users
 <ul>
-  <li> The demo must be built and run using the master branch. We currently plan to include support starting with the
-    0.13.0 release. </li>
+  <li> The demo must be built and run using the release-0.14.1 tag. </li>
   <li> Presto and Trino are not currently supported in the demo. </li>
+  <li> You will see warningss that there is no historyserver for your archittecture. You can ignore this </li>
```

Comment:
You will see `warnings` that there is no `history server` for your `architecture`. You can ignore this `.`